frontend/src/components/auth/ResetPassword.tsx

```diff
@@ -14,17 +22,23 @@ const ResetPassword = ({ email }: ResetPasswordProps) => {
     return re.test(String(emailString).toLowerCase());
   };
 
+  type ResetPassword = { ok: boolean };
+  const [resetPassword] = useMutation<{ resetPassword: ResetPassword }>(
+    RESET_PASSWORD,
+  );
+
   const onResetPasswordClick = async () => {
     if (authenticatedUser == null && !isRealEmailAvailable(email!)) {
-      alert("invalid email");
+      alert("Invalid email");
       return;
     }
+
     const resetEmail = authenticatedUser?.email ?? email;
-    if (await authAPIClient.resetPassword(resetEmail)) {
+    try {
+      await resetPassword({ variables: { email: resetEmail } });
```

Comment:
`ok: false` could come back here, and we'd still be alerting users that a reset email was sent. Can you add a check here? This should work right now:
```
const res = await resetPassword({ variables: { email: resetEmail } });
console.log(res.data?.resetPassword.ok);
```
I know we don't have an `ok: false` case right now, but I'd rather cover our bases.